melodic/distribution.yaml

```diff
@@ -1928,6 +1928,22 @@ repositories:
       url: https://github.com/ros/robot_state_publisher.git
       version: kinetic-devel
     status: maintained
+  robot_upstart:
+    doc:
+      type: git
+      url: https://github.com/clearpathrobotics/robot_upstart.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/clearpath-gbp/robot_upstart-release.git
+      version: 0.3.0-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/clearpathrobotics/robot_upstart.git
+      version: kinetic-devel
```

Comment:
@mikepurvis I'm noticing that the branch name here is a bit weird.  For Kinetic, it looks like it was `jade-devel` (which is fine if it shared code with jade), but given that this is a 'new' branch, do you mean this to be `melodic-devel`?

Reply:
Oh I should update src/doc on Kinetic then. I cut a new `kinetic-devel` branch for 0.3.0, and it's supposed to be shared between Kinetic and Melodic (at least for now).

Reply:
Makes sense.  Thanks for the explanation, I'll merge now.